projects/plugins/crm/includes/ZeroBSCRM.FormatHelpers.php

```diff
@@ -1146,6 +1148,7 @@ function zeroBSCRM_html_taskDate($task=array()){
 	    return $start_d . ' - ' . $end_d;
 	}
 
+	// phpcs:disable
```

Comment:
Why are you disabling phpcs here for the whole rest of the file?

Reply:
Oh no, that was not meant to stay there. That was left-over as I was trying to remove earlier errors to make the relevant errors easier to see. I'll clean that up in another PR.

Reply:
It looks like adding those lines in and then removing them is upsetting the linter: https://github.com/Automattic/jetpack/pull/28996
I may need to add this to a later follow-up to properly lint this file now.

Reply:
Yeah, I guess since this got merged with those in there, now removing them is seen as introducing linting errors. ☹️